src/panels/lovelace/cards/hui-picture-elements-card.js

```diff
@@ -86,12 +93,26 @@ class HuiPictureElementsCard extends LocalizeMixin(EventsMixin(PolymerElement))
           el.addEventListener('click', () => this._handleClick(entityId, element.action === 'toggle'));
           el.classList.add('clickable');
           el.title = this._computeTooltip(stateObj);
-          if (element.style) {
-            Object.keys(element.style).forEach((prop) => {
-              el.style.setProperty(prop, element.style[prop]);
-            });
-          }
           this._requiresStateObj.push({ el, entityId });
+        } else if (element.type === 'state-text') {
+          const entityId = element.entity;
+          const stateObj = this.hass.states[entityId];
+          el = document.createElement('div');
+          el.addEventListener('click', () => this._handleClick(entityId, false));
+          el.classList.add('clickable');
+          el.innerText = this._computeTextState(stateObj);
```

Comment:
We only have to initialize the elements here. No need to set the values. 1 method to create, 1 method to set values